tools/test/audio/src_test.m

```diff
@@ -435,7 +441,7 @@
 %%
 
 function test = test_defaults_src(t)
-test.comp = 'src';
+test.comp = t.comp;
```

Comment:
In function `test_run_src(test, t)`, t is unused.

Reply:
@singalsu pls fix in subsequent PR. I think this is a separate issue.

Reply:
Thanks, the fix is now in #8486.